package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "substrait",
+  "version": "0.20.0",
+  "description": "Packages Substrait protobuf files into a static module.",
```

```suggestion
  "description": "Utilities for working with Substrait plans in Javascript projects",
```
Now that the parser, etc. are part of this package I think we can update this description.

---

Made the change, thanks!